docs/source/bft_configuration.md

```diff
@@ -0,0 +1,221 @@
+* `RequestPoolSize` : The number of pending requests retained by the node. The `RequestPoolSize` is recommended to be at least double (x2) the `RequestBatchMaxCount`. This cannot be changed with a reconfiguration.
+* `RequestForwardTimeout`: Is started from the moment a request is submitted, and defines the interval after which a request is forwarded to the leader.
+* `RequestComplainTimeout`: Is started when `RequestForwardTimeout` expires, and defines the interval after which the node complains about the view leader.
+* `RequestAutoRemoveTimeout`: Is started when `RequestComplainTimeout` expires, and defines the interval after which a request is removed (dropped) from the request pool.
+* `ViewChangeResendInterval`: Defines the interval after which the ViewChange message is resent.
+* `ViewChangeTimeout`: Is started when a node first receives a quorum of `ViewChange` messages, and defines the interval after which the node will try to initiate a view change with a higher view number.
+* `LeaderHeartbeatTimeout`: Is the interval after which, if nodes do not receive a "sign of life" from the leader, they complain about the current leader and try to initiate a view change. A sign of life is either a heartbeat or a message from the leader.
+* `LeaderHeartbeatCount`: Is the number of heartbeats per `LeaderHeartbeatTimeout` that the leader should emit. The heartbeat-interval is equal to: `LeaderHeartbeatTimeout/LeaderHeartbeatCount`.
+* `CollectTimeout`: Is the interval after which the node stops listening to `StateTransferResponse` messages, stops collecting information about view metadata from remote nodes.
+
+## Reconfiguration
+
+The Smart BFT orderer supports dynamic (meaning, while the channel is being serviced) addition and removal of nodes, and configuration changes.
+The only configuration parameter which cannot be changed is the `RequestPoolSize`.
+
+Note that your cluster must be operational and able to achieve consensus before you attempt to reconfigure it.
+As a rule, you should never attempt any configuration changes to the Smart BFT consenters, such as adding or removing a consenter, or rotating a consenter's certificate, unless all consenters are online and healthy.
+
+If you do decide to change these parameters, it is recommended to only attempt such a change during a maintenance cycle. 
+Problems are most likely to occur when a reconfiguration is attempted in clusters with only a few nodes while a node is down. 
+For example, if you have four nodes in your consenters set and one of them is down, it means you have three out of four nodes alive. 
+If you extend the cluster to five nodes while in this state, you will have only three out of five nodes alive, which is not a quorum.
+The fifth node won't be able to onboard because nodes can only onboard to functioning clusters (unless the total size of the cluster less than four).
```

Comment:
I don't think this is true for BFT. We can onboard even if the cluster has no consensus because of:
```
func (c *BFTChain) Errored() <-chan struct{} {
	// TODO: Implement Errored
	return nil
}
``` 

in `chain.go`

Reply:
So what should I write instead?